DayrSuryan/DSEthiop9.xml

```diff
@@ -503,13 +507,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     The identity of  Ṗeṭros is not certain. He might be identical with
                      <persName ref="PRS7830Petros">Ṗeṭros IV</persName> of Ethiopia (in tenure 1882-1917), in this case
                      his name might have been copied from the exemplar manuscript.
-                   The original commissioners of the manuscript were probably <persName ref="PRS11960HabtaM"/> of
-                   <placeName ref="INS0352EA">ʾƎndā ʾAbbonā (Dabra Ṭǝllul)</placeName>, mentioned in
-                     supplications throughout (e.g. <locus target="#2v #53v"/>) and <persName ref="PRS12967WalattaM"/>,
-                   whose name has been erased in a supplication on
+                   The original commissioners of the manuscript were <persName ref="PRS11960HabtaM"/> (possibly a monk of
+                   <placeName ref="INS0352EA">ʾƎndā ʾAbbonā Dabra Ṭǝllul</placeName>), mentioned in
+                     supplications throughout (e.g. <locus target="#2v #53v"/>) and <persName ref="PRS12967WalattaM"/>.
+                   Her name has been erased in a supplication on
                      <locus target="#204v"/> and replaced by that of <persName ref="PRS12002Gabrami"/>.
                      <persName ref="PRS11960HabtaM"/> probably owned the manuscript and later donated it
-                     to <persName ref="PRS12002Gabrami"/> (cp. <ref target="#a1">Addition 1</ref>).
+                     to <persName ref="PRS12002Gabrami"/> (cp. <ref target="#a1">Addition 1</ref>, and <ref type="mss" corresp="DSEthiop5"/>).
```

Comment:
But here I would say, no comma before `and <ref type="mss" corresp="DSEthiop5"/>)`